src/surfaces/actionBar/actionBarActionsButtonDrawerOption.jsx

```diff
@@ -49,22 +49,22 @@ class ActionBarActionsButtonDrawerOption extends React.PureComponent {
 
         if (_.isFunction(option.onClick)) {
             if (option.disabled) {
-                return false;
+                return;
             }
 
             if (option.requiresPrompt) {
                 if (_.isFunction(onRequestPrompt)) {
                     onRequestPrompt(option);
-                    return false;
+                    return;
                 }
             }
 
-            option.onClick(onDrawerToggle);
-            return false;
+            onDrawerToggle();
+            option.onClick();
+            return;
```

Comment:
We don't need this one.  Functions automagically return when we get to the end! =)

Reply:
Actually nm.  This is not the end of the function.  Sorry; don't mind me.